src/System.Management.Automation/engine/CommandCompletion/ArgumentCompleters.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation.Language;
+
+namespace System.Management.Automation
+{
+    /// <summary>
+    /// Provides argument completion for Scope parameter.
+    /// </summary>
+    public class ScopeArgumentCompleter : IArgumentCompleter
+    {
+        private static readonly string[] scopes = new string[] { "Global", "Local", "Script" };
```

Comment:
Someday we will get https://github.com/dotnet/roslyn/pull/69820 
```suggestion
        private static ReadOnlySpan<string> s_Scopes = new string[] { "Global", "Local", "Script" };
```

Reply:
@iSazonov we will get `Field or auto-implemented property cannot be of type 'ReadOnlySpan<string>' unless it is an instance member of a ref struct.[CS8345](https://msdn.microsoft.com/query/roslyn.query?appId%3Droslyn%26k%3Dk%28CS8345%29)` if we use this like that.

Reply:
We could do `private static ReadOnlySpan<string> s_Scopes = new string[] { "Global", "Local", "Script" };` but then iteration becomes a challenge.

Reply:
Thanks for clarify!

Please rename to `s_Scopes` to follow our style guidelines.